Switch to enable profiling for db interact

ttung opened the pull request:
### Description
aspen-cli db interact --profile will print the sql statements being executed and the time spent.

### Test plan
Tried it, it works great.

phoenixAja reviewed (approved):
really nice!! would be nice to add a little comment to the documentation to let users know about this capability

ttung commented:
Added docs.

Commit:
Switch to enable profiling for db interact

### Description
aspen-cli db interact --profile will print the sql statements being executed and the time spent.

### Test plan
Tried it, it works great.
